espocrm/stack.yml

```diff
@@ -0,0 +1,60 @@
+version: '3.8'
+
+services:
+
+  mysql:
+    image: mysql:8
+    container_name: mysql
+    command: --default-authentication-plugin=mysql_native_password
```

Comment:
This line should be removed because it would break `mysql:8` (Would work with 8.0.x but not 8.4.x)